Dockerfile

```diff
@@ -29,7 +29,8 @@ ENV RUSTUP_HOME=/usr/local/rustup \
     CARGO_HOME=/usr/local/cargo \
     PATH=/usr/local/cargo/bin:$PATH
 
-RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --profile minimal
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs \
+    | sh -s -- -y --profile minimal --default-toolchain=1.63
```

Comment:
If we wanna go with this for now, could we  maybe add default `ARG` something like:
 `ARG RUST_TOOLCHAIN_VERSION=1.63.0` on top and use the variable here?

Reply:
Done 👍